app/helpers/public_body_helper.rb

```diff
@@ -40,9 +40,9 @@ def public_body_not_requestable_reasons(public_body)
   # Returns a String
   def type_of_authority(public_body)
     categories = PublicBodyCategory.
-      where(:category_tag => public_body.tag_string.split)
+      where(:category_tag => public_body.tag_string.split).to_a
 
-    types = categories.each_with_index.map do |category, index|
+    types = categories.to_a.each_with_index.map do |category, index|
```

Comment:
Why do this twice (L43 and then again on L45)? I'd just do this on L45 – closest to the `map`.